sos/plugins/samba.py

```diff
@@ -32,6 +32,7 @@ def setup(self):
         self.add_cmd_output([
             "wbinfo --domain='.' -g",
             "wbinfo --domain='.' -u",
+            "testparm -s",
             "testparm -s -v"
```

Comment:
np - generally we don't remove redundant commands from sos without some deprecation period, since some users may have a dependency on specific options/formatting (or whatever).

In this case though, I think we can probably make an exception - I really don't see much value in repeating compiled-in values and I think it's pretty unlikely anyone is doing much parsing of this data (or that if they are, adapting to just `-v` would be a huge deal for them).

If you're happy to amend & repush that'd be great - I can get this merged pretty quickly then.